sdk/python/kfp/_client.py

```diff
@@ -12,27 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import time
-import logging
+import datetime
 import json
+import logging
 import os
 import re
 import tarfile
 import tempfile
+import time
 import warnings
-import yaml
 import zipfile
-import datetime
-from typing import Mapping, Callable
+from typing import Callable, Mapping
```

Comment:
```suggestion
from typing import Callable, Mapping, Optional
```

I think you need to import Optional, which is used on row 777.

Reply:
Woops! I fixed the missing import